src/main/java/build/buildfarm/worker/shard/Worker.java

```diff
@@ -801,6 +801,7 @@ public void run() {
                     SECONDS.sleep(1);
                   }
                 } catch (InterruptedException e) {
```

Comment:
This catch does nothing in this case, it can be removed.

Reply:
Removing that catch creates a compilation error:
```
src/main/java/build/buildfarm/worker/shard/Worker.java:806: error: unreported exception InterruptedException; must be caught or declared to be thrown
```
What would you suggest? Should `InterruptedException` be rethrown as `RuntimeException`? The original implementation did not do that.

Reply:
No, it definitely should not be rethrown - Just put in a `// ignore` comment line into the InterruptedException handler, so that intention is clear.